test/integration/apiserver/apiserver_test.go

```diff
@@ -82,7 +84,7 @@ func setupWithResources(t *testing.T, groupVersions []schema.GroupVersion, resou
 	client, config, teardown := framework.StartTestServer(ctx, t, framework.TestServerSetup{
 		ModifyServerConfig: func(config *controlplane.Config) {
 			if len(groupVersions) > 0 || len(resources) > 0 {
-				resourceConfig := controlplane.DefaultAPIResourceConfigSource()
+				resourceConfig := controlplane.DefaultAPIResourceConfigSource(nil)
```

Comment:
Will this `nil` need to change if we ever used this to test `EmulationVersion`?

Reply:
`ModifyServerConfig` would only override the default resourceConfig `if len(groupVersions) > 0 || len(resources) > 0`. So no, `EmulationVersion` can be tested with the default resourceConfig, like in `TestEnableEmulationVersion`

Reply:
ack, I also see that setup and setupWithResources never provide non-empty groupVersoins/reosurces in any usages, so you are right itd never be hit. 

If we do end up enabling by default in the future, this line may need to change. but thats far away. can mark as resolved